src/shared/directives/conditional-show/services/condition-parser.service.ts

```diff
@@ -0,0 +1,202 @@
+import { _ as _score } from 'underscore';
+import peg from './condition.peg';
+import { FieldsUtils } from '../../../services/fields/fields.utils';
+
+export class ConditionParser {
+
+    private static readonly CONDITION_NOT_EQUALS = '!=';
```

Comment:
Indenting should all be 2 spaces rather than 4.

Reply:
changed it to 2